src/argilla/client/models.py

```diff
@@ -341,6 +343,8 @@ class TokenClassificationRecord(_Validators):
     search_keywords: Optional[List[str]] = None
     _span_utils: SpanUtils = PrivateAttr()
 
+    projection: Optional[bool] = None  # TODO: remove after https://github.com/argilla-io/argilla/issues/2535
```

Comment:
Some tests are failing since this new field is serialized with the rest of the fields. Since is an internal temporary value depending on how data where loaded, we can exclude it for serialization. See [here](https://docs.pydantic.dev/usage/exporting_models/#model-and-field-level-include-and-exclude)

```suggestion
    projection: Optional[bool] = Field(default=None, exclude=True)  # TODO: remove after https://github.com/argilla-io/argilla/issues/2535
```